syndesis-maven-plugin/pom.xml

```diff
@@ -38,11 +38,75 @@
     <dependency>
       <groupId>io.syndesis</groupId>
       <artifactId>model</artifactId>
+      <exclusions>
+        <exclusion>
```

Comment:
when I think about this perhaps:

```xml
<exclusion>
  <groupId>*</groupId>
  <artifactId>*</artifactId>
</exclusion>
```

might be better (I wish I did that first time around)

Reply:
Ok, changed this. But not sure why the exclusions are required anyway?

Reply:
Tried that, but are running in tons of issues: Maven duplicate plugin croaks a lot plus when running it:

```
[ERROR] Failed to execute goal io.syndesis:syndesis-maven-plugin:0.1-SNAPSHOT:generate-mapper-inspections (generate-mapper-inspections) on project runtime: Execution generate-mapper-inspections of goal io.syndesis:syndesis-maven-plugin:0.1-SNAPSHOT:generate-mapper-inspections failed: A required class was missing while executing io.syndesis:syndesis-maven-plugin:0.1-SNAPSHOT:generate-mapper-inspections: io/syndesis/core/Json
```

Reply:
So reverted it again (sorry, have to prioritize ...)